src/azure/storage/signer.rs

```diff
@@ -142,6 +161,24 @@ impl Signer {
 
     /// Apply signed results to requests.
     pub fn apply(&self, req: &mut impl SignableRequest, output: &SignedOutput) -> Result<()> {
+        self.apply_req(req, output, false)
+    }
+
+    /// Apply signed results to requests.
+    pub fn apply_sub_req(
```

Comment:
I prefer to add a new config value at `Signer` side instead of changing our public API.